src/osx/core/bitmap.cpp

```diff
@@ -949,15 +1026,18 @@ wxBitmap wxBitmap::GetSubBitmap(const wxRect &rect) const
     int destwidth = rect.width*scale ;
     int destheight = rect.height*scale ;
 
+    const wxBitmapRepresentationData* srcrep = GetBitmapData()->GetDefaultRepresentation();
+    wxBitmapRepresentationData* destrep = ret.GetBitmapData()->GetDefaultRepresentation();
```

Comment:
It's probably not feasible to apply this to all representations, but maybe worth adding a comment at least saying this.